### ISSUE TITLE
Bug: SMT_1.30_Setup.zip Doesn't work without OneDrive. but SMT_1.30.zip works fine

### ISSUE THREAD

**Issue by tylort69** [Slazanger/SMT#150]
after performing these steps the application only shows the splash screen and then closes.
https://support.microsoft.com/en-us/office/turn-off-disable-or-uninstall-onedrive-f32a17ce-3336-40fe-9c38-6efb09f944b0
and then following the steps in the drop down under "Uninstall OneDrive"

then doing both of these methods

Method 1: Disable OneDrive Integration

Press Win + R: This will open the Run dialog.
Type "gpedit.msc" and press Enter: This will open the Group Policy Editor.
Navigate to "Computer Configuration" > "Administrative Templates" > "Windows Components" > "OneDrive".
Find the policy called "Prevent the usage of OneDrive for file storage".
Double-click on the policy to open its properties.
Select "Enabled" to disable OneDrive integration.
Click "Apply" and then "OK".
Method 2: Modify the Windows Registry

Press Win + R to open the Run dialog.
Type "regedit" and press Enter to open the Windows Registry Editor.
Ctrl +F for "HKEY_CURRENT_USER\Software\Microsoft\OneDrive"
Right-click on the result and delete
Ctrl +F for "018D5C66-4533-4307-9B53-224DE2ED1FE6".
Right-click on the result ::shell(018D5C66-4533-4307-9B53-224DE2ED1FE6) and delete

**Comment by tylort69** [comment]
but SMT_1.30.zip works fine **
Only after changing the file path for the chat logs to the proper local directory

**Comment by Slazanger** [comment]
What order did you install / run SMT / disable onedrive ?  it read/writes files to My Documents\SMT  and is the same in both versions (its the same set of executables, just packaged slightly differently)...

**Comment by tylort69** [comment]
originally the order was:
Install One Drive > Install  SMT_1.30_Setup.zip and it worked
Remove one drive with the steps listed in OP and the install no longer worked.
I then tried uninstalling and reinstalling SMT_1.30_Setup.zip it still did not work.
I then tried SMT_1.30.zip and it worked fine.
I have since the OP changed the default paths in the registry for my user from One Drive paths to regular local paths and the same SMT_1.30.zip is working no issue.
